### PR TITLE
* purge compress debug

### PR DESCRIPTION
**Pull request by jayzeng**

### CODE REVIEW

File: flask_compress.py [hunk 1]
```diff
@@ -62,9 +61,6 @@ def after_request(self, response):
         else:
             app = current_app
 
-        if app.debug and not app.config['COMPRESS_DEBUG']:
-            return response
```
Comment (jayzeng) [inline flask_compress.py]: Not sure what the intention was, but the COMPRESS_DEBUG doesn't really provide any sort of debugging ability and makes using this library slightly harder. 

Feel free to close this PR if you think it is useful.

**Comment by libwilliam** [comment]
I still see value in being able to disable compression for debugging purposes (using app.debug). I could see an argument that COMPRESS_DEBUG isn't really needed though.

What do you think?

**Comment by jayzeng** [comment]
I totally agree respecting app.debug is a good idea, but having a separate debug flag while not printing out/adding any extra debugging info seems redundant in my personal opinion. What about just keeping app.debug and remove compress_debug?

**Comment by libwilliam** [comment]
I'm on board. If you don't mind tweaking your pull request to still support app.debug I will accept it. I'm not sure how I am going to handle the version number at the moment. I might bump it to 1.1.0 due to the slight API change.

Thank you for helping out the package!

**Comment by jayzeng** [comment]
More than happy to make the changes, question is, the original implementation simply returns the response if app.debug is true. Will it be more useful to do things like print added headers by this library to stdout?

**Comment by libwilliam** [comment]
Maybe, but I'm currently happy with it just returning and nothing else. Some logging could be useful in a future improvement.

**Comment by jayzeng** [comment]
OK. Updated the PR and let me know what you think

**Comment by libwilliam** [comment]
Looks good. Remove the `test_debug_default` method and I will accept the merge.

**Comment by jayzeng** [comment]
done

**Comment by libwilliam** [comment]
Sorry, I noticed one more thing. You are going to hate me by the end of this.

Keep the `test_compress_debug` method but modify it like this. That way it will still test that app.debug prevents compression.

```python
def test_compress_debug(self):
        """ Tests app.debug correctly affects response data. """
        self.app.debug = True

        response = self.client_get('/large/')
        self.assertEqual(self.large_size, len(response.data))
```

**Comment by jayzeng** [comment]
Fair enough, added the test

**Comment by libwilliam** [comment]
Merged. Thank you!

**Comment by jayzeng** [comment]
Thanks, let me know when you release a new version on pypi

**Comment by libwilliam** [comment]
PyPI is now current.

**Comment by jayzeng** [comment]
Thanks!

**Pull request merged**